wp-seo-main.php

```diff
@@ -287,6 +287,13 @@ function wpseo_init() {
 
 	// Init it here because the filter must be present on the frontend as well or it won't work in the customizer.
 	new WPSEO_Customizer();
+
+	// Initializes the link watcher.
+	$storage           = new WPSEO_Link_Storage();
```

Comment:
Please put this inside a class which initializes these classes instead of putting them all inside the main plugin file.

Reply:
Please add a comment why this needs to be loaded here.